app/routes.py

```diff
@@ -1,2 +1,213 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.task import Task 
+from app.models.goal import Goal 
+from sqlalchemy import desc
+from sqlalchemy import asc
+import datetime
+import requests 
+import os
+from dotenv import load_dotenv
+
+tasks_bp = Blueprint("tasks" , __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def return_response_task(task_id):
+    task = Task.query.get(task_id)
+    return {
+            "id": task.task_id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete" : False if task.completed_at == None else True 
+        }
+def return_response_goal(goal_id):
+    goal = Goal.query.get(goal_id)
+    return {
+            "id": goal.goal_id,
+            "title": goal.title
+            }
+
+def return_response_goal_id(task_id):
+    task = Task.query.get(task_id)
+    return {
+            "id": task.task_id,
+            "title": task.title,
+            "goal_id": task.goal_id,
+            "description": task.description,
+            "is_complete": False if task.completed_at == None else True 
+            }
```

Comment:
Awesome that you created these helper functions. You could also move these to each model and clean up the routes file. In the model, they would become instance methods and you would be putting `self.task_id` instead